### DISCUSSION TITLE
Добавить свойство Enable к лидерборду

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#22] [Ideas]
Добавить свойство Enable у лидерборда, чтобы при нажатии нельзя было отрыть сам лидерборд

**Comment by kkohno** [comment]
а как тогда смотреть какие команды? и пример применения и зачем?

**Reply by KukushkaSex** [reply]
Пусть сразу в команду игроков добавлять будут... Да и это поможет с многим, например тот же режим дезран где он не нужер
